### PR TITLE
Button changes

### PR DESCRIPTION
**Pull request by dckesler**

### CODE REVIEW

**Comment by keithhalterman** [comment]
🍠

**Comment by keithhalterman** [comment]
👞

File: src/multi-selector.js [hunk 1]
```diff
@@ -365,11 +365,14 @@ const MultiSelector = React.createClass({
...
 							<span className="cps-error-block">{this.props.invalidMsg}</span>}
 					</div>
-					<div className="cpr-multi-selector__dialog__items">
+					<div
+						style={this.props.dialogHeight ? {maxHeight: this.props.dialogHeight} : {}}
+						className="cpr-multi-selector__dialog__items">
 						{this.getSearchItems(this.props.items)}
-						{this.state.searchValue &&
+					</div>
+						{(this.state.searchValue && this.props.doneButton) &&
```
Comment (leahjlou) [inline src/multi-selector.js:373]: Why do you need a `searchValue` to see the done button?

Comment (dckesler) [inline src/multi-selector.js:373]: Twas as I was instructed.

**Comment by leahjlou** [comment]
🍉

**Pull request merged**